server/README.md

```diff
@@ -0,0 +1,32 @@
+## App Engine web-proxy
```

Comment:
TODO: improve doc with deploy script when https://passculture.atlassian.net/browse/PC-14035 is DONE